config.h

```diff
@@ -149,6 +151,7 @@ void pv_config_set_creds_secret(char *secret);
 void pv_config_set_debug_shell(bool shell);
 void pv_config_set_debug_shell_autologin(bool autologin);
 void pv_config_set_debug_ssh(bool ssh);
+void pv_config_set_debug_shell_active(bool shell);
```

Comment:
Sorry for not seeing this before. Usually, we store session variables somewhere under the pv struct or as a static, but config are for user set values. There are some exceptions but they have their own defined level, and no level fits with this, which is changed by pantavisor policy. The one that you are using (ARGS) is for arguments that can be set to the pantavisor command (see init.c).

In this case, it would be weird from user perspective if pvcontrol conf ls would return something other than what it was configured by the user.

For me, it is ok to keep maybe some bool PV_DEBUG_SHELL instead of PV_DEBUG_SHELL_ACTIVE, so user can disable/enable this feature. However, session on/off should go in pv struct or somewhere else.